draft-ietf-trans-rfc6962-bis.md

```diff
@@ -1838,10 +1852,11 @@ It is up to a client's local policy to specify the quantity and form of
 evidence (SCTs, inclusion proofs or a combination) needed to achieve
 compliance and how to handle non-compliance.
 
-A TLS client can only evaluate compliance if it has given the TLS server the
+A TLS client can only determine compliance if it has given the TLS server the
```

Comment:
The long-held WG consensus is that 6962-bis should not specify "client behaviour" (see slides 8 and 9 of https://datatracker.ietf.org/meeting/91/materials/slides-91-trans-4), except where necessary to participate in the CT protocol.  That's why we chose the verb "evaluate" rather than anything stronger.

Alexey suggested changing "evaluate" to "enforce", but I think that sounds a bit too strong.

Aren't "evaluate" and "determine" synonymous?  I think "evaluate" was just fine.

Reply:
I agree "enforce" is the wrong word to use.  To me, "evaluate" implies making a decision and taking action on it while "determine" is missing that action step. But I might be wrong, and will change it back.